dockerunit-core/src/main/java/com/github/qzagarese/dockerunit/internal/lifecycle/DockerUnitAfter.java

```diff
@@ -24,7 +24,6 @@ public void evaluate() throws Throwable {
 		try {
 			statement.evaluate();
 		} catch(Throwable t) {
-			t.printStackTrace();
 			throw t;
```

Comment:
if this is going to only re-throw there is no need for the `catch`

Reply:
:+1: